app/input_objects/pages/question_input.rb

```diff
@@ -6,10 +6,14 @@ class Pages::QuestionInput < BaseInput
   # TODO: We could lose these attributes once we have an Check your answers page
   attr_accessor :answer_settings, :page_heading, :guidance_markdown
 
+  # Only used for error summary
+  attr_accessor :selection_options
```

Comment:
I think if we don't need to be able to assign this attribute we can use `attr_reader` to stop ruby generating a method to assign it:
```suggestion
  attr_reader :selection_options
```

Reply:
(I don't think we've been hugely consisten about this elsewhere though)